docs/website/docs/examples/qdrant_zendesk/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: Similarity Searching with Qdrant
+description: Learn how to use the dlt source, Zendesk and dlt destination, Qdrant to conduct a similarity search on your tickets data.
+keywords: [similarity search, example]
+---
+import Header from '../_examples-qdrant-header.md';
+
+<Header
+    intro="In this tutorial, you will learn how to do use dlt to store your
+    vectorized Zendesk tickets data in the dlt destination; Qdrant. You can
+    use Qdrant's vectorization and similarity searching capabilities on your tickets data,
+    while using dlt as a medium to automate your pipeline."
+    slug="qdrant_zendesk"
+    run_file="qdrant-snippets"
+/>
+
+This article outlines a system to map vectorized ticket data from Zendesk to Qdrant, similar to our guide on the topic concerning [Weaviate](https://dlthub.com/docs/dlt-ecosystem/destinations/qdrant). In this example, we will:
+- Connect to our Zendesk source.
```

Comment:
add links to our documentation:
- Connect to our [Zendesk source](https://dlthub.com/docs/dlt-ecosystem/verified-sources/zendesk)
- ...
- [Create a dlt pipeline](https://dlthub.com/docs/walkthroughs/create-a-pipeline) with Qdrant as destination.

and so on